Description:
 In Ubuntu, Use gksu in gddccontrol.desktop to get access to the DDC/CI device.
Origin: Ubuntu
Author: Emmet Hikory <emmet.hikory@gmail.com>
Forwarded: no
---
 src/gddccontrol/gddccontrol.desktop.in |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- ddccontrol-0.4.2.orig/src/gddccontrol/gddccontrol.desktop.in
+++ ddccontrol-0.4.2/src/gddccontrol/gddccontrol.desktop.in
@@ -2,7 +2,7 @@
 Encoding=UTF-8
 Name=Monitor Settings
 Comment=Set your monitor preferences (contrast, brightness,...) and manage your profiles
-Exec=gddccontrol
+Exec=gksu gddccontrol
 Icon=gddccontrol
 Terminal=false
 Type=Application
